avalon/pipeline.py

```diff
@@ -1269,16 +1268,14 @@ def path_from_config():
             log.debug("Template references unavailable data: %s" % e)
             return None
 
-        if os.path.exists(path):
-            return os.path.normpath(path)
+        return os.path.normpath(os.path.dirname(path))
```

Comment:
@iLLiCiTiT noticed that this is actually returning only the folder and not the full path. That will break all the loading won't it?

Reply:
O.o Good spot !
Looks like something happened while merging #371, but wasn't able to figure out how it changed to return dir path.